akka-http-core/src/test/scala/akka/http/impl/engine/parsing/HttpHeaderParserSpec.scala

```diff
@@ -33,10 +33,10 @@ abstract class HttpHeaderParserSpec(mode: String, newLine: String) extends AkkaS
       check {
         s"""nodes: 0/H, 0/e, 0/l, 0/l, 0/o, 1/Ω
            |branchData:${" " /* explicit trailing space */ }
-           |values: 'Hello""" -> parser.formatRawTrie
+           |values: Symbol(Hello)""" -> parser.formatRawTrie
```

Comment:
Would that still work with older Scala versions?

Reply:
Ah, of course it wouldn't.

We could:

```suggestion
           |values: ${Symbol(Hello)}""" -> parser.formatRawTrie
```

... but at that point perhaps it'd be better to replace the symbols with regular strings here - WDYT?

Reply:
Yes, good point. I don't think anything speaks against using strings here.